rskj-core/src/test/java/co/rsk/peg/federation/PowpegMigrationTest.java

```diff
@@ -225,7 +225,6 @@ private void testChangePowpeg(
         for (UTXO utxo : existingUtxos) {
             assertTrue(utxosToMigrate.stream().anyMatch(storedUtxo -> storedUtxo.equals(utxo)));
         }
-        assertTrue(federationStorageProvider.getNewFederationBtcUTXOs(btcParams, activations).isEmpty());
 
         // Trying to create a new powpeg again should fail
         // -2 corresponds to a new powpeg was elected and the Bridge is waiting for this new powpeg to activate
```

Comment:
I guess tests will come in different prs, like we are doing with other classes. Right?

Reply:
yes, i created a task for fixing this test and another one for adding tests